ai-engineering/create.mdx

```diff
@@ -1,133 +1,159 @@
+
+## Build your capability
+
+Define your capability using your framework of choice. Here’s an example using Vercel's [AI SDK](https://ai-sdk.dev/), which includes [many examples](https://sdk.vercel.ai/examples) covering different capability design patterns. Popular alternatives like [Mastra](https://mastra.ai) also exist.
+
+```ts src/lib/capabilities/classify-ticket.ts expandable
+import { generateObject } from 'ai';
+import { openai } from '@ai-sdk/openai';
+import { wrapAISDKModel } from 'axiom/ai';
+import { z } from 'zod';
+
+export async function classifyTicket(input: { 
+  subject?: string; 
+  content: string 
+}) {
+  const result = await generateObject({
+    model: wrapAISDKModel(openai('gpt-4o-mini')),
+    messages: [
+      {
+        role: 'system',
+        content: 'Classify support tickets as: question, bug_report, or feature_request.',
+      },
+      {
+        role: 'user',
+        content: input.subject 
+          ? `Subject: ${input.subject}\n\n${input.content}` 
+          : input.content,
+      },
+    ],
+    schema: z.object({
+      category: z.enum(['question', 'bug_report', 'feature_request']),
+      confidence: z.number().min(0).max(1),
+    }),
+  });
 
-### Define a capability as a prompt object
+  return result.object;
+}
+```
 
-In Axiom AI engineering, every capability is represented by a `Prompt` object. This object serves as the single source of truth for the capability’s logic, including its messages, metadata, and the schema for its arguments.
+The `wrapAISDKModel` function instruments your model calls for Axiom’s observability features. Learn more in the [Observe](/ai-engineering/observe) section.
 
-For now, these `Prompt` objects can be defined and managed as TypeScript files within your own project repository.
+## Gather reference examples
 
-A typical `Prompt` object looks like this:
+As you prototype, collect examples of inputs and their correct outputs.
+
+```ts
+const referenceExamples = [
+  {
+    input: { 
+      subject: 'How do I reset my password?',
+      content: 'I forgot my password and need help.' 
+    },
+    expected: { category: 'question' },
+  },
+  {
+    input: { 
+      subject: 'App crashes on startup',
+      content: 'The app immediately crashes when I open it.' 
+    },
+    expected: { category: 'bug_report' },
+  },
+];
+```
 
-```ts /src/prompts/email-summarizer.prompt.ts
+These become your ground truth for evaluation. Learn more in the [Evaluate](/ai-engineering/evaluate/overview) section.
+
+## Structured prompt management
+
+<Note>
+The features below are experimental. Axiom’s current focus is on the evaluation and observability stages of the AI engineering workflow.
+</Note>
+
+For teams wanting more structure around prompt definitions, Axiom’s SDK includes experimental utilities for managing prompts as versioned objects.
+
+### Define prompts as objects
+
+Represent capabilities as structured `Prompt` objects:
+
+```ts src/prompts/ticket-classifier.prompt.ts
```

Comment:
IMO: completely remove prompt management from docs for now. I would like to stop exporting the current version of it from the SDK as well.